### ISSUE TITLE
Search button magnifying glass needs label.

### ISSUE THREAD

**Issue by alisonrae** [PublicisSapient/accessible-ecommerce-demo#145]
In the global navigation menu, provide a label for the button element in the search bar. Currently there is no text and the screen reader just announces, "button". Call it "search".

Do not use this code:
```
<button class="global-header__search-btn">
      <svg class="icon icon--search-btn">
        <use xlink:href="../sprite.svg#search"></use>
      </svg>
    </button>
```

Use something like this to make it accessible:
```
<button class="global-header__search-btn">
<img class="icon icon--search-btn" alt="Search" src="../sprite.svg#search">
    </button>
```

**Comment by aevanson** [comment]
@alisonrae as an alternative to your proposed solution, can we instead use `<button aria-label="search" >...`? I've set up an "SVG sprite sheet", which requires the use of `svg>use` and doesn't work with `<img>` tags.

**Comment by alisonrae** [comment]
Can you make a span tag with the label that is hidden by CSS? I don't want to have empty buttons that rely on aria to work.

**Comment by brianelton** [comment]
Fix on public site
